scavenger-api/build.gradle.kts

```diff
@@ -55,7 +55,7 @@ configure<DependencyManagementExtension> {
 }
 
 kotlin {
-    jvmToolchain(11)
+    jvmToolchain(17)
 }
 
 tasks.withType<KotlinCompile> {
```

Comment:
maybe remove this option.

Reply:
If we remove this option, it seems cannot detect jvm target like belows.
> w: Inconsistent JVM-target compatibility detected for tasks 'compileJava' (17) and 'compileKotlin' (1.8).

Reply:
Have you tried deleting all build output, including model, entity, etc... and retrying?

Reply:
I deleted all the modules and retried, but they also crashed.
I think it would be better to mark it explicitly because there is a reason as below and there are cases where each modules use different versions of jdk.

> When there is no explicit information about the `jvmTarget` value in the build script, its default value is `null`, and the compiler translates it to the default value `1.8`
> https://kotlinlang.org/docs/gradle-configure-project.html#what-can-go-wrong-if-targets-are-incompatible

Reply:
I think it would be better keep this option to run the service as before.
spring boot 3.x guide still includes `JSR305` option.

> https://github.com/spring-projects/spring-boot/blob/3.2.x/spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/kotlin.adoc#null-safety